UnitsNet.Tests/CustomCode/VolumeTests.cs

```diff
@@ -106,6 +106,11 @@ public class VolumeTests : VolumeTestsBase
 
         protected override double DecalitersInOneCubicMeter => 1e2;
 
+        /// <summary>
+        /// https://www.legislation.gov.uk/uksi/1995/1804/made
+        /// </summary>
+        protected override double ImperialQuartsInOneCubicMeter => 879.876993196;
```

Comment:
Verified.

Weird, wolfram normally is very precise, but seems to round here.
https://www.wolframalpha.com/input?i=1+cubic+meter+in+imperial+quarts